### PR TITLE
[Idenyt.dk] Update ruleset.

### PR DESCRIPTION
**Pull request by bardiharborow**

### CODE REVIEW

File: src/chrome/content/rules/Idenyt.dk.xml [hunk 1]
```diff
@@ -1,20 +1,10 @@
-<!--
-	Cloudflare SSL on ^ and www
-
-	Problematic domains:
-		^ ²
-		www ²
-		foto ¹
-		jul2k8 ¹
-		(www.)tv ¹
```
Comment (J0WI) [inline src/chrome/content/rules/Idenyt.dk.xml]: Why did you remove all comments? Some are still true.

Comment (bardiharborow) [reply src/chrome/content/rules/Idenyt.dk.xml]: @J0WI not sure actually, fixed.
